homeassistant/components/daikin/switch.py

```diff
@@ -72,3 +83,50 @@ async def async_turn_on(self, **kwargs):
     async def async_turn_off(self, **kwargs):
         """Turn the zone off."""
         await self._api.device.set_zone(self._zone_id, "0")
+
+
+class DaikinStreamerSwitch(ToggleEntity):
```

Comment:
Please inherit from `SwitchEntity` instead of `ToggleEntity`. Sorry I missed that the first review. Switch platform entitiessshould alwaysiinherit `SwitchEntity`.

Reply:
Ok. Most projects I work on prefer clean history. I'll try to remember this when working on HA.

Switched to SwitchEntity. I copied ToggleEntity from the DaikinZoneSwitch class above.

Reply:
We squash when we merge so the final history will be clean. 👍